embedding/cpp/include/flutter_app.h

```diff
@@ -111,9 +111,27 @@ class FlutterApp : public flutter::PluginRegistry {
 
   // The user-defined pixel ratio.
   //
-  //  Defaults to the device pixel ratio if the value is 0.
+  // Defaults to the device pixel ratio if the value is 0.
   double user_pixel_ratio_ = 0.0;
 
+  // Whether the app should be supported pointing device(mouse) or not.
```

Comment:
```suggestion
  // Whether the app should support pointing devices (mouse).
```
Please check other comments, too.

Reply:
done